src/utils/config.ts

```diff
@@ -13,6 +13,8 @@ const ciConfig = {
   commitMessage: process.env.CI_COMMIT_MESSAGE,
   forgeType: process.env.CI_FORGE_TYPE,
   githubToken: process.env.PLUGIN_GITHUB_TOKEN,
+  giteaToken: process.env.PLUGIN_GITEA_TOKEN,
+  giteaUrl: process.env.PLUGIN_GITEA_URL,
```

Comment:
CI_FORGE_URL should be enouth as default

Reply:
Yep, that simplifies the config. (Quite an old PR already 😅 ).